ndonnx/_data_types/aliases.py

```diff
@@ -55,3 +61,52 @@
 nuint32: NUInt32 = NUInt32()
 nuint64: NUInt64 = NUInt64()
 nutf8: NUtf8 = NUtf8()
+
+
+_canonical_names = {
```

Comment:
I think `_canonical_names` and _kinds` should rather life with the `DType` objects, but that can be something for the future.